### ISSUE TITLE
`servePath` and `inngest:dev` url mismatch

### ISSUE THREAD

**Issue by Tobbe** [inngest/envelop-plugin-inngest#180]
The `serve()` function defaults to `./redwood/functions/inngest`
```
export const handler = serve(inngest, inngestFunctions, {
  servePath: '/.redwood/functions/inngest',
```

The `inngest:dev` command defaults to `http://localhost:8911/inngest`. 
They should match for the dev server to know where to connect.

**Comment by dthyresson** [comment]
Good catch. Also, the docs/readme needs to be updated to note that they must be the same.
